contrib/utilities/indent

```diff
@@ -140,8 +140,10 @@ find tests include source benchmarks cookbooks unit_tests \
   | xargs -P 10 -I {} bash -c "indent_file {}"
 
 
+SOURCE_FILES=`find include source unit_tests cookbooks benchmarks doc \( -name '*.cc' -o -name '*.h' -o -name '*.prm' -o -name '*.h.in' -o -name 'CMakeLists.txt' -o -name '*.cmake' -o -name '*.md' -o -wholename '*modules/changes/*' \) -print`
```

Comment:
Sadly, this doesn't work with ``-print0`` in bash, which would be cleaner and safer (special characters in filenames).